<!DOCTYPE html>
<html lang="en">

<!-- Head -->

<head>        
    <!-- Metadata, OpenGraph and Schema.org -->
    

    <!-- Standard metadata -->
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <title>Aadharsh Aadhithya A | publications</title>
    <meta name="author" content="Aadharsh  Aadhithya A" />
    <meta name="description" content="publications by categories in reversed chronological order. generated by jekyll-scholar." />
    <meta name="keywords" content="aadharsh-aadhithya,aadharsh,aadhithya,undergraduate-reasearcher" />


    <!-- Bootstrap & MDB -->
    <link href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha256-DF7Zhf293AJxJNTmh5zhoYYIMs2oXitRfBjY+9L//AY=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous" />

    <!-- Fonts & Icons -->
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.15.4/css/all.min.css" integrity="sha256-mUZM63G8m73Mcidfrv5E+Y61y7a12O5mW4ezU3bxqW4=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/academicons@1.9.1/css/academicons.min.css" integrity="sha256-i1+4qU2G2860dGGIOJscdC30s9beBXjFfzjWLjBRsBg=" crossorigin="anonymous">
    <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons">

    <!-- Code Syntax Highlighting -->
    <link rel="stylesheet" href="https://gitcdn.link/repo/jwarby/jekyll-pygments-themes/master/github.css" />

    <!-- Styles -->
    <link rel="icon" href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 viewBox=%220 0 100 100%22><text y=%22.9em%22 font-size=%2290%22>üëãüèª</text></svg>">
    <link rel="stylesheet" href="/assets/css/main.css">
    <link rel="canonical" href="https://aadharshaadhithya.github.io/publications/">

    <!-- Dark Mode -->
    <script src="/assets/js/theme.js"></script>
    <script src="/assets/js/dark_mode.js"></script>
</head>

<!-- Body -->

<body
  class="fixed-top-nav sticky-bottom-footer">

  <!-- Header -->
    <header>

      <!-- Nav Bar -->
      <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top">
        <div class="container">
          <a class="navbar-brand title font-weight-lighter" href="https://aadharshaadhithya.github.io/">Aadharsh Aadhithya A</a>
          <!-- Navbar Toggle -->
          <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
            <span class="sr-only">Toggle navigation</span>
            <span class="icon-bar top-bar"></span>
            <span class="icon-bar middle-bar"></span>
            <span class="icon-bar bottom-bar"></span>
          </button>

          <div class="collapse navbar-collapse text-right" id="navbarNav">
            <ul class="navbar-nav ml-auto flex-nowrap">

              <!-- About -->
              <li class="nav-item ">
                <a class="nav-link" href="/">about</a>
              </li>
              
              <!-- Blog -->
              <li class="nav-item ">
                <a class="nav-link" href="/blog/">blog</a>
              </li>

              <!-- Other pages -->
              <li class="nav-item ">
                <a class="nav-link" href="/projects/">projects</a>
              </li>
              <li class="nav-item active">
                <a class="nav-link" href="/publications/">publications<span class="sr-only">(current)</span></a>
              </li>

              <!-- Toogle theme mode -->
              <div class="toggle-container">
                <a id="light-toggle">
                  <i class="fas fa-moon"></i>
                  <i class="fas fa-sun"></i>
                </a>
              </div>
            </ul>
          </div>
        </div>
      </nav>
    </header>

  <!-- Content -->
  <div class="container mt-5">
    <!-- page.html -->
        <div class="post">

          <header class="post-header">
            <h1 class="post-title">publications</h1>
            <p class="post-description">publications by categories in reversed chronological order. generated by jekyll-scholar.</p>
          </header>

          <article>
            <!-- _pages/publications.md -->
<div class="publications">
  <h2 class="year">2021</h2>
  <ol class="bibliography"><li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"><abbr class="badge">ICMSS</abbr></div>

        <!-- Entry bib key -->
        <div id="9673636" class="col-sm-8">
        
          <!-- Title -->
          <div class="title">Implementation of Hack ALU using Quantum Dot Cellular Automata</div>
          <!-- Author -->
          <div class="author">Aadhithya, A. Aadharsh,¬†Akshaya, J.,¬†Devi, K. Ghaayathri,¬†Nithesh, B.,¬†Lal, G. Jyothish,¬†and Soman, K.P.
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>2021 Fourth International Conference on Microelectronics, Signals &amp; Systems (ICMSS)</em> 2021
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="https://ieeexplore.ieee.org/document/9673636" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Quantum Dot Cellular Automata (QCA) technology is one of the emerging next-generation nano-scale technologies, to subdue the limitations of existing CMOS technologies. As researchers continue to work hard to find an alternative to CMOS technology, QCA provides a solution for a faster computer with a smaller size and low power consumption. This paper implements an operation-rich ‚ÄúHACK ALU‚Äù, capable of performing many operations. Hack AL U is a versatile design that implements 64 op-erations with just 6 control bits along with novel implementations of adders and multiplexers. This ALU has been designed using a bottom-up approach by beginning the design by constructing the basic gates and progressing to the construction of adders, and multiplexers. The scope of the research is to provide an efficient ALU design that can be integrated in the CPU design. The present work has been implemented and tested on the QCA designer software. Experimental evaluation shows that logical and arithmetic operations are consistent in the proposed design.</p>
          </div>
        </div>
      </div>
</li></ol>

  <h2 class="year">2023</h2>
  <ol class="bibliography">
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"><abbr class="badge">NFMD</abbr></div>

        <!-- Entry bib key -->
        <div id="NFMD" class="col-sm-8">
        
          <!-- Title -->
          <div class="title">Prospects of NFMD for Power System Frequency
and Amplitude Estimation</div>
          <!-- Author -->
          <div class="author">Aadhithya, A Aadharsh.,¬†Mohan, Neethu.,¬†Kumar, S Sachin.,¬†K.P., Soman,¬†and Poornachandran, Prabhaharan.
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>7th IEEE International Conference on
Recent Advances and Innovations in Engineering</em> 2023
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="" class="btn btn-sm z-depth-0" role="button">HTML</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p> Power quality monitoring and parameter estima-
tion are essential for the proper functioning of modern power
grids. Various techniques have been proposed to estimate the
characteristics of power signals and to gain insight into their
dynamics. Non-stationary Fourier mode decomposition (NFMD)
is a new interpretable time-frequency analysis framework for
non-stationary and nonlinear signals. This work investigates
the prospects of NFMD in the estimation of power-system
characteristics such as fundamental frequency, amplitude and
presence of disturbance components. Usage of NFMD as a noise
removal technique is also explored in the current work. Results
show that NFMD is robust to noisy disturbances and can perform
well in noisy environments. The proposed methodology can be
used for real-time analysis and interpretation of large volume of
data from advanced smart metering techniques such as micro-
phasor measurement units in microgrids</p>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"><abbr class="badge">GMd</abbr></div>

        <!-- Entry bib key -->
        <div id="GMd" class="col-sm-8">
        
          <!-- Title -->
          <div class="title"> Exploiting Graph Matrix Duality for Efficient Graph Data Processing</div>
          <!-- Author -->
          <div class="author">Aadhithya, A Aadharsh.,¬†Rejathalal, Vinith.,¬†K.P., Soman,¬†and Poornachandran, Prabhaharan
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>7th IEEE International Conference on
Recent Advances and Innovations in Engineering</em> 2023
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="" class="btn btn-sm z-depth-0" role="button">HTML</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p> 
   Linear algebra is the backbone of modern applied data analytics. Not only does it provide the theoretical substratum for the development of analytical algorithms it also lends the technological support for extremely efficient implementations. The linear algebra ecosystem consists of remarkably fast hardware support with equally efficient software solutions leveraging the hardware capacity. Graphs are an ubiquitous modelling framework for the representation and analysis of connected data.A flipside from practical perspective is that conventional graph algorithms are challenging as they are hard to parallelize and often make too much use of non-linear data structures which do not have hardware support. In this context, this paper presents an invitation to an alternate way to look at graph processing in terms of algebraic constructs, formulations and technology. The paper presents the theoretical underpinnings of the new view and also some interesting progresses at the frontier level in this direction. Above all, an intended objective is to present a compelling invite to the academia and research community to adopt this approach in their academic and research activities.
  </p>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"><abbr class="badge">CGR</abbr></div>

        <!-- Entry bib key -->
        <div id="CGR" class="col-sm-8">
        
          <!-- Title -->
          <div class="title">Genus Level Classification of SARS-COV-2 using Chaos Game Representation</div>
          <!-- Author -->
          <div class="author">
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em></em> 2023
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="" class="btn btn-sm z-depth-0" role="button">HTML</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p> Disruptions due to novel pathogens are inevitable. The recent pandemic due to SARS-CoV-2 is an indispensable example of this very fact. This paper aims in providing a genus-level idea of a novel pathogen of its plausible genetic origins. In this paper, several Machine Learning techniques are used for the classification of SARS-CoV-2 at the genus level. This shall also give ideas on the origins of SARS-CoV-2.</p>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"><abbr class="badge">MCR</abbr></div>

        <!-- Entry bib key -->
        <div id="MCR" class="col-sm-8">
        
          <!-- Title -->
          <div class="title">Optimal Perfect Phylogeny Generation using ILP and NMF</div>
          <!-- Author -->
          <div class="author">
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em></em> 2023
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="" class="btn btn-sm z-depth-0" role="button">HTML</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p> 
  The concept of Perfect Phylogeny is observed to be non-universal but in some very specific studies they have shown provide
great insights from a biological standpoint. Most natural phlogenies are imperfect but this is debated by data noise acting as a
disguise, the solution to uncover the underlying perfect phylogeny in the logically simplest approach is the Minimum Character
Removal(MCR) problem. Anther variant of the solution achieved by change in perspective and computational methodologies is
the Maximal Character Compatibility(MCC) problem. Both the MCR and MCC problems are solved optimally with reasonable
efficiency by employing the Integer Linear Programming(ILP) technique. A central part of MCC solution involves solving the
Max-clique problem of the generated representation allowing numerous clique solving algorithms to generate various solutions.
The comparison between solutions optimality and run-time of these methods substantiate the goal of the different algorithms being
applied. The above methods are formulated, and implemented using modern programming languages like python and matlab. This
exploration introduces the problem of perfect phylogeny, its ILP formulations and its solution along with comparisons between
different methods in consideration
  </p>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"><abbr class="badge">MCP</abbr></div>

        <!-- Entry bib key -->
        <div id="MCP" class="col-sm-8">
        
          <!-- Title -->
          <div class="title">Comparitive Study of Continuous
Approximations and ILP for Maximal
Clique Problems</div>
          <!-- Author -->
          <div class="author">
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em></em> 2023
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="" class="btn btn-sm z-depth-0" role="button">HTML</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p> 
  The maximal clique problem has been a cornerstone for developments in combinatorial
optimization. The problem also shows up in innumerable practically relevant scenarios. With rapid
advancements in Integer Linear Programming(ILP) solvers, it becomes necessary to have timely benchmark
studies against state of the art methods to check if ILP solvers are ready for practical use cases. This work
presents a benchmark study between ILP, BK algorithm and few other approximate methods with standard
DIMACS instances. This work also aims in providing a friendly introduction to clique based graph problems
and one of its application in biological contexts, prediction of missing edges in a protein-protein interaction
network. This paper also outlines in supplementary section, simple methods to solve clique problems using
excel,python and matlab.
  </p>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"><abbr class="badge">RNA</abbr></div>

        <!-- Entry bib key -->
        <div id="RNA" class="col-sm-8">
        
          <!-- Title -->
          <div class="title">Unraveling the folds of RNA using ILP</div>
          <!-- Author -->
          <div class="author">
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em></em> 2023
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="" class="btn btn-sm z-depth-0" role="button">HTML</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p> 
  Almost all of the problems in bioinformatics are usually solved using dynamic programming and a few using the graph algorithms, However, in current trends a variety of modern biological and bioinformatics problems are deciphered using Computational methods, this calls for the need for a multidisciplinary interventions in bioinformatics for advancements in Modern Biology.The brand-new methodologies of applying Computational Algorithms like Integer linear programming will completely transform the fields of modern biology forever. In this work, a simple attempt has been made to bind the fields of Bioinformatics with Integer Linear Programming. This paper focuses on the RNA folding problem giving insights about the reason for RNA folding, a detailed description about the various components or loops present in the folded structure of an RNA and the formulation of RNA folding as an ILP problem.
  </p>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"><abbr class="badge">Motif</abbr></div>

        <!-- Entry bib key -->
        <div id="Motif" class="col-sm-8">
        
          <!-- Title -->
          <div class="title">Finding Network Motifs: A comparative study between ILP and Symmetric Rank-One NMF 
Clique Problems</div>
          <!-- Author -->
          <div class="author">
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em></em> 2023
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="" class="btn btn-sm z-depth-0" role="button">HTML</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p> 
  </p>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"><abbr class="badge">GrbI</abbr></div>

        <!-- Entry bib key -->
        <div id="GrbI" class="col-sm-8">
        
          <!-- Title -->
          <div class="title">An Informal Introduction to Semirings for GraphBLAS using Matlab </div>
          <!-- Author -->
          <div class="author">
          </div>

          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em></em> 2023
          </div>
        
          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="" class="btn btn-sm z-depth-0" role="button">HTML</a>
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p> 
  </p>
          </div>
        </div>
      </div>
</li>
</ol>


</div>

          </article>

        </div>

  </div>

  <!-- Footer -->    <footer class="sticky-bottom mt-5">
      <div class="container">
        ¬© Copyright 2023 Aadharsh  Aadhithya A. 
      </div>
    </footer>

  <!-- JavaScripts -->
  <!-- jQuery -->
  <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script>

  <!-- Bootsrap & MDB scripts -->
  <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.11.2/dist/umd/popper.min.js" integrity="sha256-l/1pMF/+J4TThfgARS6KwWrk/egwuVvhRzfLAMQ6Ds4=" crossorigin="anonymous"></script>
  <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/js/bootstrap.min.js" integrity="sha256-SyTu6CwrfOhaznYZPoolVw2rxoY7lKYKQvqbtqN93HI=" crossorigin="anonymous"></script>
  <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script>

  <!-- Mansory & imagesLoaded -->
  <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script>
  <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script>
  <script defer src="/assets/js/mansory.js" type="text/javascript"></script>
  
  <!-- Medium Zoom JS -->
  <script src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.6/dist/medium-zoom.min.js" integrity="sha256-EdPgYcPk/IIrw7FYeuJQexva49pVRZNmt3LculEr7zM=" crossorigin="anonymous"></script>
  <script src="/assets/js/zoom.js"></script><!-- Load Common JS -->
  <script src="/assets/js/common.js"></script>

  <!-- MathJax -->
  <script type="text/javascript">
    window.MathJax = {
      tex: {
        tags: 'ams'
      }
    };
  </script>
  <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script>
  <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>

  


</body>

</html>
